torch/nn/modules/module.py

```diff
@@ -629,6 +629,13 @@ def _load_from_state_dict(self, state_dict, prefix, strict, missing_keys, unexpe
             key = prefix + name
             if key in state_dict:
                 input_param = state_dict[key]
+
+                if input_param.shape != param.shape:
+                    # local shape should match the one in checkpoint
+                    error_msgs.append('Size mismatch: copying a param of {} from checkpoint, '
```

Comment:
I would add a `continue` here. Otherwise the same key may add another error message below.

Reply:
Do you mean the message in line 645? when copying a different size, this exception was not triggered. Do you know when will that be throwing an exception?
I didn't add continue bc it will list all mismatched keys in one shot, like 'weight' & 'bias'.

Reply:
It won't trigger if the shapes broadcastable, which is why we need this check here. But if they are not broadcastable, it will trigger, and then we have two error messages. :) 

I'm not sure that I understand the point about all mismatched keys. Could you give an example?

Reply:
Yea that makes sense. 
For all mismatched keys, actually after a second thought it still works with adding `continue`. I was thinking about eg. your are trying to copy {'weight':size(1,2), 'bias':(1,4)} to {'weight':size(2,2), 'bias':(2,4)}, the error message will have two lines listing both keys.
I will get another one liner to add the `continue`. Thanks! :D

Reply:
Sounds great. :)